dataactbroker/handlers/submission_handler.py

```diff
@@ -600,15 +600,34 @@ def certify_dabs_submission(submission, file_manager):
         if window.block_certification:
             return JsonResponse.error(ValueError(window.message), StatusCode.CLIENT_ERROR)
 
-    # check revalidation threshold
     sess = GlobalDB.db().session
+    # Check revalidation threshold
     reval_thresh = get_revalidation_threshold()['revalidation_threshold']
     if reval_thresh and reval_thresh >= get_last_validated_date(submission.submission_id):
         return JsonResponse.error(ValueError("This submission has not been validated since before the revalidation "
                                              "threshold ({}), it must be revalidated before certifying.".
                                              format(reval_thresh.replace('T', ' '))),
                                   StatusCode.CLIENT_ERROR)
 
+    # Get the year/quarter of the submission and filter by them
+    sub_quarter = submission.reporting_fiscal_period // 3
+    sub_year = submission.reporting_fiscal_year
+    quarter_reval = sess.query(QuarterlyRevalidationThreshold).filter_by(year=sub_year, quarter=sub_quarter).\
+        one_or_none()
+
+    # If we don't have a quarterly revalidation threshold for this year/quarter, they can't submit
+    if not quarter_reval:
+        return JsonResponse.error(ValueError("No submission window for this year and quarter was found. If this is an "
+                                             "error, please contact the Service Desk."), StatusCode.CLIENT_ERROR)
+
+    # Make sure everything was last validated after the start of the submission window
+    last_validated = datetime.strptime(get_last_validated_date(submission.submission_id), '%Y-%m-%dT%H:%M:%S')
```

Comment:
Optional/preferential: `get_last_validated_date(submission.submission_id)` can be stored as a variable that can be used in line 606 and here as well.